tests/negative/files/preexisting_nodes.go

```diff
@@ -0,0 +1,279 @@
+// Copyright 2018 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package files
+
+import (
+	"github.com/coreos/ignition/tests/register"
+	"github.com/coreos/ignition/tests/types"
+)
+
+func init() {
+	register.Register(register.NegativeTest, ForceFileCreation())
+	register.Register(register.NegativeTest, ForceDirCreation())
+	register.Register(register.NegativeTest, ForceLinkCreation())
+	register.Register(register.NegativeTest, ForceHardLinkCreation())
+
+	register.Register(register.NegativeTest, ForceFileCreationOverNonemptyDir())
+}
+
+func ForceFileCreation() types.Test {
+	name := "Force File Creation"
+	in := types.GetBaseDisk()
+	out := types.GetBaseDisk()
+	config := `{
+	  "ignition": { "version": "2.2.0-experimental" },
+	  "storage": {
+	    "files": [{
+	      "filesystem": "root",
+	      "path": "/foo/bar",
+	      "contents": {
+	        "source": "http://127.0.0.1:8080/contents"
+	      },
+		  "overwrite": false
+	    }]
+	  }
+	}`
+	in[0].Partitions.AddFiles("ROOT", []types.File{
+		{
+			Node: types.Node{
+				Directory: "foo",
+				Name:      "bar",
+			},
+		},
+	})
+	out[0].Partitions.AddFiles("ROOT", []types.File{
```

Comment:
As these are negative tests specifying an output partition should be unnecessary.